packages/terra-core-docs/src/terra-dev-site/doc/button/AnchorButton.doc.mdx

```diff
@@ -0,0 +1,53 @@
+import { Badge } from 'terra-button/package.json?dev-site-package';
+
+import ButtonBlock from './example/ButtonBlock?dev-site-example';
+import ButtonAnchor from './example/ButtonAnchor?dev-site-example';
+import ButtonCompact from './example/ButtonCompact?dev-site-example';
+import ButtonDisabled from './example/ButtonDisabled?dev-site-example';
+import ButtonOnClick from './example/ButtonOnClick?dev-site-example';
+import ButtonVariant from './example/ButtonVariant?dev-site-example';
```

Comment:
You need to import only the `ButtonAnchor` here since all other examples are not displayed on this doc page.

Reply:
Changed in https://github.com/cerner/terra-core/pull/3655/commits/76622fb8fb70fa44335e5c714ef5511c3b1579e6